src/sentry/static/sentry/app/views/settings/components/forms/model.jsx

```diff
@@ -244,14 +244,7 @@ class FormModel {
   doApiRequest({apiEndpoint, apiMethod, data}) {
     const endpoint = apiEndpoint || this.options.apiEndpoint;
     const method = apiMethod || this.options.apiMethod;
-    return new Promise((resolve, reject) => {
-      this.api.request(endpoint, {
-        method,
-        data,
-        success: response => resolve(response),
-        error: error => reject(error),
```

Comment:
I think the error handling is a bit different. `requestPromise()` receives an errorobject as its parameter, while `request`'s `options.error` gets the response as the first argument. Are there any error callbacks used with this function?

Reply:
Not sure, but looks like it fails tests, so I won't worry about this right now.